stitchclient/client.py

```diff
@@ -112,7 +112,7 @@ def moving_average_bytes_per_record(self):
             num_records += stats.num_records
             num_bytes += stats.num_bytes
 
-        return num_bytes // num_records
+        return num_bytes // num_records if num_records else None
```

Comment:
I kind of think this method _should_ throw a `ZeroDivisionError` when `num_records == 0`. Bytes per record is a nonsense statistic if there aren't any records, and returning `None` might mask that. If it's exceptional then clients of this class are forced to think about why they're taking the moving average of no records. I might even take that a step further to do something like:

```python
if num_records == 0:
   raise Exception("Can't take moving average: num_records == 0")

return num_bytes // num_records
```

I was wondering how numpy handled this case -- it seems it will return `numpy.nan` and log a warning:

```
In [2]: np.array([]).mean()
RuntimeWarning: Mean of empty slice.
RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Out[2]: nan
```

I'm also a little concerned about modifying the behavior of this method -- it doesn't have the `_` prefix so maybe downstream projects are relying on it (and possibly relying on it raising).